package.json

```diff
@@ -32,6 +32,7 @@
   },
   "scripts": {
     "lint": "eslint .",
+    "prepare": "webpack",
```

Comment:
Is there a reason to call this "prepare"? Maybe we can use "build" to be a little more conventional?

```suggestion
    "build": "webpack",
```

Reply:
npm runs `prepare` by default when you publish a package. I can keep `build` but just have `prepare` run that.